css/style.css

```diff
@@ -747,3 +747,122 @@ body {
   bottom: 40px;
   font-size: 20px;
 }
+
+.cards .contact {
+  width: 375px;
+  height: 779px;
+  background-image: url(../jalal.PNG);
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin-top: -150px;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+
+.contact .me {
+  color: #ffff;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  margin-top: 105px;
+  font-family: 'poppins', sans-serif;
+}
+
+.contact p {
+  width: 327px;
+  height: 137px;
+  font-family: 'poppins1', sans-serif;
+  color: #ebebff;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  margin: 0 auto;
+  margin-top: -20px;
+}
+
+.contact_form {
+  display: flex;
+  margin: auto;
+  flex-direction: column;
+  margin-top: 45px;
+  width: 327px;
+}
+
+.contact_form .form1 {
+  padding: 15px;
+  border-radius: 8px;
+  border: none;
+  outline: none;
+  font-family: 'poppins1', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
+}
+
+.contact_form .form2 {
+  padding: 15px;
+  border-radius: 8px;
+  border: none;
+  outline: none;
+  margin-top: 17px;
+  font-family: 'poppins1', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
+}
+
+.contact_form .form3 {
+  padding: 15px;
+  border-radius: 8px;
+  border: none;
+  outline: none;
+  margin-top: 17px;
+  padding-bottom: 140px;
+  font-family: 'poppins1', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #cfd8dc;
+}
```

Comment:
- [ ] [OPTIONAL] Nice work so far 👍🏿 i notice the textarea is hard to read as it is very transparent, to fix the issue I kindly suggest you the color style. 😃